src/frontend/screens/Settings/CreateOrJoinProject/CreateProject/ProjectCreated.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import {defineMessages, useIntl} from 'react-intl';
+import {BackHandler, StyleSheet, View} from 'react-native';
+import GreenCheck from '../../../../images/GreenCheck.svg';
+import {NativeRootNavigationProps} from '../../../../sharedTypes';
+import {Text} from '../../../../sharedComponents/Text';
+import {Button} from '../../../../sharedComponents/Button';
+import {CommonActions, useFocusEffect} from '@react-navigation/native';
+
+const m = defineMessages({
+  projectCreated: {
+    id: 'screens.Settings.CreateOrJoinProject.ProjectCreated.projectCreated',
+    defaultMessage: '{projectName} Created!',
+  },
+  inviteDevice: {
+    id: 'screens.Settings.CreateOrJoinProject.ProjectCreated.invitedDevice',
+    defaultMessage: 'Invite Device',
+  },
+  goToMap: {
+    id: 'screens.Settings.CreateOrJoinProject.ProjectCreated.goToMap',
+    defaultMessage: 'Go to Map',
+  },
+});
+
+export const ProjectCreated = ({
+  route,
+  navigation,
+}: NativeRootNavigationProps<'ProjectCreated'>) => {
+  const {formatMessage: t} = useIntl();
+
+  // disables back button
+  useFocusEffect(
+    React.useCallback(() => {
+      const subscription = BackHandler.addEventListener(
+        'hardwareBackPress',
+        () => true,
+      );
+
+      return () => subscription.remove();
+    }, []),
+  );
```

Comment:
probably should have a PR that adds a shared hook for this 😄

Reply:
yeah....